pro_tes/tasks/tasks/track_task_progress.py

```diff
@@ -0,0 +1,91 @@
+# """Celery background task to process task asynchronously."""
+
+
+from pro_tes.utils.db_utils import DbDocumentConnector
+import logging
+from time import sleep
+from typing import (
+    Dict,
+)
+
+from foca.database.register_mongodb import _create_mongo_client
+from foca.models.config import Config
+from flask import (Flask, current_app)
+
+from pro_tes.ga4gh.tes.models import (
+    TesState
+)
+
+import tes
+from pro_tes.celery_worker import celery
+from pro_tes.ga4gh.tes.states import States
+logger = logging.getLogger(__name__)
```

Comment:
Please mind the order of imports:

1. Imports from built-in modules
2. Imports from third party packages
3. Imports from _this_ package

Please order each group alphabetically by the name of the module/package. And please leave one empty line between each block.

Also, please leave one line between the imports and any following statements, like the logger initialization.

You can check other modules for comparison, as well as the official PEP 8 Python style guide and the PEP on docstrings. (Note that these don't prescribe ordering of modules/packages within each of the three blocks, but I find it improves readability and maintainability)

Reply:
Thanks. Imports from this module aren't actually sorted alphabetically by module, but okay, not a big problem. Just try to stick to it in the future. It's certainly helpful if you have a lot of imports.